src/decaylanguage/modeling/ampgen2goofit.py

```diff
@@ -87,3 +87,74 @@ def ampgen2goofit(filename, ret_output=False):
     if ret_output:
         return output.getvalue()
     return None
+
+
+def ampgen2goofitpy(filename, ret_output=False):
```

Comment:
Maybe add docstrings where appropriate and also update the file header docstring?

Reply:
Thanks a lot.